application/src/main/java/org/opentripplanner/apis/vectortiles/DebugStyleSpec.java

```diff
@@ -260,6 +263,35 @@ private static List<StyleBuilder> vertices(VectorSourceLayer vertices) {
     );
   }
 
+  private static List<StyleBuilder> elevators(VectorSourceLayer edges, VectorSourceLayer vertices) {
+    return List.of(
+      StyleBuilder.ofId("elevator-hop-edge")
+        .group(ELEVATORS_GROUP)
+        .typeLine()
+        .vectorSourceLayer(edges)
+        .edgeFilter(ElevatorHopEdge.class)
+        .lineColor(ORANGE)
+        .lineWidth(LINE_HALF_WIDTH)
+        .lineOffset(LINE_OFFSET)
+        .minZoom(6)
+        .maxZoom(MAX_ZOOM)
+        .intiallyHidden(),
```

Comment:
These are slightly problematic as they are used for all elevators, but only visible for the horizontal elevators currently since the line has 0 length usually. I was thinking should we rename this so it's more clear that these will only show up for the horizontal elevators or somehow draw them as circles when they have 0 length and as lines when they don't. However, it's also possible that we don't do anything about it now.